test/activate-event.html

```diff
@@ -130,6 +130,17 @@
         assert.equal(s.selected, '0');
       });
 
+      test('activates after detach and re-attach', function() {
+        // Detach and re-attach
+        var parent = s.parentNode;
+        parent.removeChild(s);
+        parent.appendChild(s);
+        
+        // select Item 2
```

Comment:
don't fix this now, but this capitalization kills me :sob: (at least Select should be capitalized to match the comment 5 lines up)